manifests/init.pp

```diff
@@ -28,15 +28,20 @@
 #
 class caddy (
 
-  String                         $install_path    = $caddy::params::install_path,
-  String                         $caddy_user      = $caddy::params::caddy_user,
-  String                         $caddy_group     = $caddy::params::caddy_group,
-  String                         $caddy_log_dir   = $caddy::params::caddy_log_dir,
-  String                         $caddy_tmp_dir   = $caddy::params::caddy_tmp_dir,
-  String                         $caddy_home      = $caddy::params::caddy_home,
-  Enum['personal', 'commercial'] $caddy_license   = 'personal',
-  Enum['on','off']               $caddy_telemetry = 'off',
-  String                         $caddy_features  = 'http.filter,http.git,http.ipfilter'
+  String                         $install_path     = $caddy::params::install_path,
+  String                         $caddy_user       = $caddy::params::caddy_user,
+  String                         $caddy_group      = $caddy::params::caddy_group,
+  String                         $caddy_log_dir    = $caddy::params::caddy_log_dir,
+  String                         $caddy_tmp_dir    = $caddy::params::caddy_tmp_dir,
+  String                         $caddy_ssl_dir    = $caddy::params::caddy_ssl_dir,
+  String                         $caddy_home       = $caddy::params::caddy_home,
+  Enum['personal', 'commercial'] $caddy_license    = 'personal',
+  String                         $caddy_account_id = '',
```

Comment:
Instead of empty string, can you default to undef?
```suggestion
   Optional[String[1]]           $caddy_account_id,
```

Same applies for `$caddy_api_key` We defined that some time ago in our [review guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr).